javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,13 @@
+/* exported filterOutStrings */
+// empty array for not strings
+// start at index 0
+// if value at index is a number, retrun to the array
+function filterOutStrings(values) {
+  var x = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This doesn't filter out strings, it just finds numbers. What if there was other data types mixed in such as booleans?